libs/ng-strapi-auth/src/lib/core/services/auth/auth.service.ts

```diff
@@ -37,20 +38,40 @@ export class AuthService {
     }
 
     login<T = DefaultUserModel>(username: string, password: string): Observable<UserModel<T>> {
-        return this.httpClient.post<any>(`${this.authApiBase}/auth/local`, { identifier: username, password: password })
-            .pipe(map((response: { jwt: string, user: UserModel<T> }) => {
-                if (response.jwt && response.user && response.user.blocked == false) {
-                    if (this.allowedRoles.includes(response.user.role.type)) {
-                        localStorage.setItem(LocalStorageKeyEnum.CURRENT_JWT, response.jwt);
-                        this.authUserChanged$.next(response.user);
-                        this.authToken = localStorage.getItem(LocalStorageKeyEnum.CURRENT_JWT)!;
-                    } else {
-                        this.snackbarService.showSnackBar(SnackBarTypeEnum.ERROR, 'Forbidden')
-                    }
-                }
-                return response.user;
-            }));
+      let response: AuthResponseModel<T>;
```

Comment:
No need to state the whole AuthResponse here, just jwt as string should be enough no ?